src/components/Footer/Footer.jsx

```diff
@@ -30,86 +147,28 @@ const Footer = () => {
-              If your organization is interested in partnering with PASS and would like to discuss
-              further, contact us below.
-            </Typography>
-            <Button variant="contained" color="secondary" sx={{ my: '1rem' }}>
-              Partnership Proposal
-            </Button>
-          </Grid>
-          <Grid item xs={2}>
-            <Typography color={`${theme.palette.tertiary.main}`}>PASS LOGO</Typography>
-            <Typography color="#fff" mb={5}>
-              tagline
-            </Typography>
-            <Typography color={`${theme.palette.tertiary.main}`}>Follow Us</Typography>
-            <Link href="https://twitter.com/" target="_blank" rel="noopener" color="#fff" mr={1}>
-              <TwitterIcon />
-            </Link>
-            <Link href="https://www.facebook.com/" target="_blank" rel="noopener" color="#fff">
-              <FacebookIcon />
-            </Link>
-            <Link
-              href="https://www.instagram.com/"
-              target="_blank"
-              rel="noopener"
-              color="#fff"
-              ml={1}
-            >
-              <InstagramIcon />
-            </Link>
-          </Grid>
-          <Grid item xs={2}>
-            <Typography color={`${theme.palette.tertiary.main}`}>Built By:</Typography>
-            <Link
-              href="https://www.codeforpdx.org/"
-              target="_blank"
-              rel="noopener"
-              underline="none"
-            >
-              <Typography variant="body2" color="#fff">
-                C4PDX LOGO
-              </Typography>
-            </Link>
-            <Typography variant="body2" color={`${theme.palette.tertiary.main}`} mb={5}>
-              ©{new Date().getFullYear()}{' '}
-              <Link
-                href="https://www.codeforpdx.org/"
-                target="_blank"
-                rel="noopener"
-                underline="none"
-                color={`${theme.palette.tertiary.main}`}
-              >
-                Code for PDX
-              </Link>
-            </Typography>
-            <Typography variant="body2">
-              <Link href="https://www.codeforpdx.org/" underline="none" color="#fff">
-                Privacy Policy
-              </Link>
-            </Typography>
-            <Typography variant="body2">
-              <Link href="https://www.codeforpdx.org/" underline="none" color="#fff">
-                Terms and Conditions
-              </Link>
-            </Typography>
-          </Grid>
-        </Grid>
+      <Container maxWidth={isSmallScreen ? 'md' : 'lg'}>
+        <Stack
+          alignItems="center"
+          direction={isReallySmallScreen ? 'column' : 'row'}
+          spacing={isSmallScreen ? 1 : 4}
+          divider={
+            <Divider
+              orientation={isReallySmallScreen ? 'horizontal' : 'vertical'}
+              flexItem={isReallySmallScreen ? null : 'flexItem'}
```

Comment:
Just checked this attribute for Divider on MUI. It looks like it suppose to just be a boolean for true/false. I think you can change this to `flexItem={isReallySmallScreen ? null : true}`

Reply:
Switched to your suggestion.